presto-main/src/main/java/com/facebook/presto/operator/AggregationOperator.java

```diff
@@ -176,7 +177,16 @@ public Page getOutput()
         }
 
         state = State.FINISHED;
-        return pageBuilder.build();
+
+        Page page = pageBuilder.build();
+
+        // prepend the row type block for partial outputs
+        if (partial) {
+            BlockBuilder blockBuilder = TINYINT.createBlockBuilder(null, 1);
```

Comment:
A future commit introduces a `RowType` enum. Is there any reason that change is not part of this commit?